src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -234,25 +321,26 @@ private void move(Vector spdVec, double rot) {
    * @param rateLimit whether or not to use slew rate limiting
    */
   private void move(Vector spdVec, double rot, boolean rateLimit) {
-    Vector spdCommanded = spdVec;
     m_currentRotationRadians = rot;
 
+    spdCommanded.setX(spdVec.x());
```

Comment:
> Isn't there a copy-constructor for Vector? I won't block you on it but that'd be ideal.

This isn't copy construction, this is moving one vector into another. In many libraries it'd be called `copyTo()` or `move()`. It moves the contents to an already allocated buffer.  I told them they could merge since and we can move to address it later.

Reply:
Yeah, I won't block on it either but we should open a task to track this.